recipes/fast-dds/config.yml

```diff
@@ -1,5 +1,8 @@
 versions:
   "2.3.2":
-    folder: all
+    folder: 2.3.X
   "2.3.3":
-    folder: all
+    folder: 2.3.X
+  "2.3.4":
+    folder: 2.3.X
```

Comment:
What is the rational?

Reply:
in next PR want to add 2.4.X and 2.5.X and they differ in some regions

Reply:
Are those _regions_ too big? Sometimes it is better to have some _big_ `if/else` that to have several `conanfile.py` files that will start to diverge. Do you already have a draft for those changes?

Reply:
@jgsogo i will check it again and rechange it back to all if not too different.

Reply:
@jgsogo i found my local 2.4.X again - the difference indeed is truly just related to the patches - i will look again other conanfiles how to handle that. 2.4.X dont need patches 2.3.X needs. guess i should check the version before apply the patches inside conanfile - if that is the proper way

Reply:
ok renamed back to all